radicle-cli/src/commands/patch/create.rs

```diff
@@ -33,11 +35,18 @@ blank is also okay.
 -->
 "#;
 
+#[inline]
+fn confirm<D: fmt::Display>(prompt: D, options: &Options) -> bool {
+    !options.confirm || term::confirm(prompt)
+}
```

Comment:
Yeah maybe you're right. Just feels weird to pass options into this function as well, but we can live with it. I think it might be worth having a prompter that takes a boolean and then knows whether to prompt or not, eg.
```rust
let prompt = Prompter::new(options.confirm);
if prompt.confirm("Are you sure?") {
...
}
```

Could be a nice re-usable component if we use this pattern a lot.

Reply:
What do you think about something like this for now?

https://github.com/radicle-dev/heartwood/pull/214/commits/9d8e3e3bfddf178cf4db103a3dcf87ddd21717df

Reply:
no no no, this is very arcane

Reply:
I'd leave it how you have it for now

Reply:
Yeah, objects with methods are more common and idiomatic if you need to return something from a function